### PR TITLE
Add required fields to job cluster samples

### PR DESCRIPTION
**Pull request by JaeguKim**
I found `flinkoperator_v1beta1_flinkjobcluster.yaml` and `flinkoperator_v1beta1_remotejobjar.yaml` missed required fields for v1beta1 `flinkclusters.flinkoperator.k8s.io`.

### CODE REVIEW

File: config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ spec:
   image:
     name: flink:1.14.2
   jobManager:
+    accessScope: Cluster
```
Comment (regadas) [inline config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml:24]: Hi @JaeguKim Thanks for this! Which version are you using? I believe this was addressed and shouldn't be required now with the latest beta.

Comment (JaeguKim) [reply config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml:24]: I'm using [flink-cluster-crd.yaml](https://github.com/spotify/flink-on-k8s-operator/blob/master/helm-chart/flink-operator/templates/flink-cluster-crd.yaml) which is latest beta I believe.   
It looks like the [accessScope](https://github.com/spotify/flink-on-k8s-operator/blob/1967e49314bf15e4cc433c90ca51d1ae6114d5a2/helm-chart/flink-operator/templates/flink-cluster-crd.yaml#L3884) is still required field of `jobManager` property and [restartPolicy](https://github.com/spotify/flink-on-k8s-operator/blob/1967e49314bf15e4cc433c90ca51d1ae6114d5a2/helm-chart/flink-operator/templates/flink-cluster-crd.yaml#L1590) is also still required field of `job` property.

Comment (regadas) [reply config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml:24]: ah I see what's going on, helm CRD is outdated.

**Review by regadas — APPROVED** [review]
Gonna go ahead and merge this. the accessScope with the latest CRD in #323 should not be required but doesn't heart having it here.

**Comment by regadas** [comment]
Thanks for this @JaeguKim sorry the delay.

**Pull request merged**